Source/Extensions/Blazorise.DataGrid/CellEditContext.cs

```diff
@@ -22,9 +23,20 @@ public class CellEditContext
     /// <typeparam name="TItem"></typeparam>
     public class CellEditContext<TItem> : CellEditContext
     {
-        public CellEditContext( TItem item )
+        /// <summary>
+        /// Method that will be called when cell is manually updated.
+        /// </summary>
+        private readonly Action<string, object> CellUpdated;
```

Comment:
Do you do private fields with capital?

Reply:
Only when they're methods, events or delegates